File: fizzbuzz.py
for i in range(100):
    if i % 3 == 0 and i % 5 == 0:
        print("FizzBuzz")
    if i % 3 == 0:
        print("Fizz")
    if i % 5 == 0:
        print("Buzz")
    else:
         print(i)

File: fizzbuzz.js
console.log("Bonjour");
let result;
for (let i = 0; i <= 100; i++){
  result = i;
    if (i % 3 == 0){
        result = "Fizz";
    }
    if (i % 5 == 0){
        result = "Buzz";
    }
    if (i % 3 == 0 && i % 5 == 0){
        result = "FizzBuzz";
    }

    console.log(result);
}
